sections/annoyances.txt

```diff
@@ -216,6 +225,7 @@ life.hu,reblog.hu,ok.hu,videa.hu###accept-cookie
 origo.hu##.mw-allow-notification-dialog
 origo.hu##.newsletter-layer
 origo.hu##div.facebook-layer-box
+otpbank.hu##.cdk-overlay-container:has(ib-bs-recommended-analytics-consent)
```

Comment:
does this rule need the CSS rule that is currently in another section?

Reply:
yes. for some reason i thought they get merged in the end. will move this rule to both files